update-and-show-snapshots.js

```diff
@@ -0,0 +1,24 @@
+let { exec } = require('child_process')
+let { promisify } = require('util')
+let chalk = require('chalk')
+
+let showSnapshots = require('./show-snapshots')
+
+let execCommand = promisify(exec)
+
+async function updateAndShowSnaphots (print, error, cwd, filter) {
+  print(chalk.blue('\nUpdating snapshots... \n'))
+
+  let { stdout, stderr } = await execCommand('"./node_modules/.bin/jest" -u', {
+    cwd
+  })
+
+  print(chalk.grey(stdout))
```

Comment:
Here you will lose Jest color output. We need to keep origin colors, because `jest -u` can fall and colors will help us to find the reasons.

Reply:
In Node.js we have `std: inherit` option to bind executed script to existed `stdout`/`stderr`. In this case you will not loose colors.